app/views/discovered_hosts/_discovered_hosts_list.html.erb

```diff
@@ -38,8 +38,39 @@
       <td class="hidden-tablet hidden-xs"><%= host.primary_interface.try(:subnet) %></td>
       <td class="hidden-tablet hidden-xs"><%= disc_report_column(host) %></td>
       <td>
+        <!-- Modal -->
```

Comment:
The comment is out of indentation.

One more thing, I realized that the modal dialog is being rendered per each individual host. It looks like it's all same for all hosts, can you create just one instance of it and little bit of javascript that sets required things?